server/routes/home.js

```diff
@@ -385,8 +385,20 @@ module.exports = function Index({
       const user = await userService.getUser(res.locals)
       const details = await offendersService.getOffenderDetails(res.locals, bookingId)
 
```

Comment:
We just need to block the race condition that one person starts a categorisation and another arrives here to "flag" the offender